README.md

```diff
@@ -1,6 +1,6 @@
 # Commodore Component: csi-cloudscale
 
-This is a [Commodore][commodore] Component for [csi-cloudscale](https://github.com/cloudscale-ch/csi-cloudscale).
+This is a [Commodore][commodore] Component for csi-cloudscale.
```

Comment:
Why remove the link?

Reply:
that wasn't intentional, fixed in the latest version.